js/src/wp-seo-modal.js

```diff
@@ -0,0 +1,128 @@
+/* global yoastModalConfig */
```

Comment:
We should reduce this to a single react instance, that renders all modals Preferably also use a single IntlProvider but this could cause conflicts instead we namespace translations per modal.

Reply:
Also it would no longer be needed to specify a specific mountNode, cleaning up the dom and the PHP side code a bit.

Reply:
Not sure we're speaking about the same thing, but what if I want a first modal with the X close button and then a second modal without it but with the Close button at the end? Or the first one with the heading, the second one without? The whole thing is designed to have this level of flexibility.